docs/source/onedal/algorithms/clustering/kmeans-init.rst

```diff
@@ -47,6 +47,34 @@ Computing method: *dense*
 The method chooses first :math:`k` feature vectors from the training set
 :math:`X`.
 
+.. _kmeans_init_c_math_random_dense:
+
+Computing method: *random_dense*
+--------------------------------
+
+The method chooses random :math:`k` feature vectors from the training set
+:math:`X`.
+
+.. _kmeans_init_c_math_plus_plus_dense:
+
+Computing method: *plus_plus_dense* (only on CPU)
+-------------------------------------------------
+
+The method designed as follows: the first centroid :math:`c_0` is selected randomly and :math:`C = \{ c_1 \}`. Then do
```

Comment:
```suggestion
The method is designed as follows: the first centroid :math:`c_0` is selected randomly, and :math:`C = \{ c_1 \}`. Then do
```

Reply:
The "Then do" part is not clear to me. At first, it seems as if you talk about the concept of the method, about something that users should not do by themselves (The method is designed as follows).  But then there is some call to action. Do users do all the steps, including the one mentioned in the first sentence?

Reply:
you are right, it was not written well. changed, ty